dist/platforms/ubuntu/entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Ensure machine ID is randomized for personal license activation
+if [[ "$UNITY_SERIAL" = F* ]]; then
+  echo "Randomizing machine ID for personal license activation"
+  dbus-uuidgen > /etc/machine-id && mkdir -p /var/lib/dbus/ && ln -sf /etc/machine-id /var/lib/dbus/machine-id
+fi
```

Comment:
@AndrewKahr Hi, may I ask why this is done only for the personal license?

We are maintaining an internal fork of this repo that needed machine id randomization, but we are using Pro licenses.

Reply:
Correct me if I'm wrong, but running say 5 test jobs in parallel with the same machine id might be problematic cause all the jobs would have the same license activation.

We randomize the machine id when running with a pro license and saw flaky activation errors disappearing. Our hypothesis is that having the same machine id leads to having the same "activation" across different parallel runs, and when one run returns the license the others can no longer use the activation they think they have.

Btw, I've written about this in the help channel on Discord as well. I'm happy to help with any other information I can.

Reply:
Originally I had it randomize for all licenses but immediately on release we got a lot of bug reports for failed activations so it got reverted and likely will stay that way for the time being.